cfgov/v1/templates/v1/images_report.html

```diff
@@ -0,0 +1,73 @@
+{% extends 'wagtailadmin/reports/base_report.html' %}
+{% load i18n wagtailadmin_tags %}
+
+{% block actions %}
+    {% if view.list_export %}
+        <div class="actionbutton">
+            <a href="{{ view.csv_export_url }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a>
+        </div>
+    {% endif %}
+{% endblock %}
+
+{% block results %}
+    {% if object_list %}
+        <table class="listing">
+            <thead>
+                <tr>
+                    <th>
+                        Title
+                    </th>
+                    <th>
+                        File
+                    </th>
+                    <th>
+                        Size (bytes)
+                    </th>
+                    <th>
+                        Collection
+                    </th>
+                    <th>
+                        Tags
+                    </th>
+                    <th>
+                        Uploaded on
+                    </th>
+                    <th>
+                        Uploaded by
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for image in object_list %}
+                    <tr>
+                        <td>
+                            {{ image.title }}
+                        </td>
+                        <td>
+                            <a href="{{ image.url }}">
+                                {{ image.url }}
+                            </a>
```

Comment:
This might not make sense for images, but in the documents report I [combined the document's filename and URL into a single column](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/templates/v1/documents_report.html#L50-L52). That's mostly because Wagtail annoyingly clips instead of scrolls tables that are too big horizontally, but also because I didn't get much extra information from a separate URL field in the Wagtail view for documents. The URL field is there in the documents CSV, where it serves more of a purpose. Not sure if that's a good pattern for the image report or not, though.